gelic-idealab/impress
# Komodo v0.5.6 Integrations

# Komodo v0.5.6 Integrations

Target release: v0.1.2 alpha

## Capture Button in Instructor Menu

A new instructor-only menu is available in the settings panel (in spectator mode only, so that VR users do not accidentally activate it).

It has a single button that can start and stop captures for the current session.

The button, for the time being, does not synchronize its state to other users. 

Please see https://github.com/gelic-idealab/komodo-unity/pull/93 for a more detailed description of the new changes. 

## Bug fix: Color Picker Initialization

This bug has not been present in Impress lately, but it does occur whenever trying to directly integrate the KomodoCore package into the Impress project. It has been manually fixed this time, and the changes will hopefully be upstreamed to komodo-unity soon. For convenience, the bug is detailed below.

### Type of change


### How Has This Been Tested?

Actions to take: 

1. Open the Create menu
2. Press the Draw button 
3. Start drawing

Expected result: 

1. Color of the stroke matches the selected color

Buggy result: 

1. Color of the stroke is either black or white
2. It's required to close and re-open the draw tab to restore the proper color selection

_REPLACE ME: any relevant details for your test configuration_



[Duplicate all of the above for each distinct feature / bug fix / etc.]
_____

# Test Configuration

* _REPLACE ME: Browser vendor and version: e.g., Chrome Version 91.0.4472.164 (Official Build) (64-bit)_
* _REPLACE ME: VR Device and OS version: e.g., Oculus Quest v29_
- [ ] PC VR (Link / AirLink)

# Checklist:


Modified from this article:
Phillip Johnston, “A GitHub Pull Request Template for Your Projects - Embedded Artistry,” Embedded Artistry, Aug. 04, 2017. https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/ (accessed Jul. 22, 2021).
